### ISSUE TITLE
New Component: Search

### ISSUE THREAD

**Issue by melloware** [primefaces-extensions/primefaces-extensions#129]
Original PF issue: https://github.com/primefaces/primefaces/issues/396

What about a new component which can markup any occurences of a word or a part of a word in a specified object(div, panel,...)? Same thing like the browser search function, but for specified containers. This would be great, especially in addition to filter in dataTable.

eg.
```xml
<p:mark styleClass="styleClassForMarkDown" 
                        for=":form:searchContainer" 
                        value="wordToSearch" />
```

This jquery component would be ideal: https://markjs.io/

**Comment by jxmai** [comment]
@melloware If this ticket is still relevant, I will review and see if it can be implemented.

**Comment by melloware** [comment]
Yes still relevant I just hadn't had the use case for it to implement!
